public/rulesets/default/2_3_3_Response_body

```diff
@@ -0,0 +1,16 @@
+rules:	
+  - id: "2.3.4"
+    title: "Response Body JSON/BSON Encapsulation"
+    message: "The response body must support JSON or BSON encapsulation. If a body is present, the Content-Type must be application/json;charset=UTF-8 or application/bson."
+    option: Mandatory
+    location: paths
+    element:
+      - responses
+    call: {
```

Comment:
BSON type won't be supported in first version as we don't have such API examples.
for the JSON type, please add the requirements that under content there must be "application/json" key:
```
      responses:
        description: Association object that needs to be created
        content:
          application/json:
```